x-pack/plugins/siem/public/components/timeline/body/column_headers/default_headers.ts

```diff
@@ -18,7 +18,7 @@ export const defaultHeaders: ColumnHeader[] = [
   },
   {
     columnHeaderType: defaultColumnHeaderType,
-    id: 'event.severity',
+    id: 'message',
```

Comment:
Should we display a tooltip when you hover over the `message` field, since its length will almost always exceed the default column width?

Reply:
Yea, I updated it with it and added unit tests for it.

<img width="342" alt="Screen Shot 2019-06-05 at 9 47 56 PM" src="https://user-images.githubusercontent.com/1151048/59006400-ecfcf100-87de-11e9-9376-f5c20504b131.png">